src/main/java/uk/gov/hmcts/reform/sscs/util/OtherPartyDataUtil.java

```diff
@@ -225,4 +225,32 @@ public static boolean hasNewOtherPartyAdded(List<CcdValue<OtherParty>> before, L
         }
         return false;
     }
+
+    public static void validateOtherPartyForSscs5Case(SscsCaseData sscsCaseData) {
```

Comment:
bit strange that this method is called validate, but it doesn't validate anything